addons/snailmail/static/src/components/message/message.js

```diff
@@ -1,34 +1,23 @@
 odoo.define('snailmail/static/src/components/message/message.js', function (require) {
 'use strict';
 
-const { Message } = require('@mail/components/message/message');
```

Comment:
no matter what (replace or override) you should always require the original file to guarantee it is called first, otherwise it just works by luck